### PR TITLE
Simplify some stuff

### PR DESCRIPTION
**Pull request by Swifticul**

### CODE REVIEW

File: guides/create-account.md [hunk 1]
```diff
@@ -1,30 +1,31 @@
...
+The recommended way to use SideStore is with a burner Apple ID Account (to avoid temporarily losing an important account to log-in's falsely flagged as malicious).
 
-## Requires
+## Requirements
 
-- Emailadres (you can use: https://temp-mail.org/)
-- Phone number (use one you can actually keep recieving SMS codes for)
-- An iDevice (To avoid: This action cannot be completed at this time (-22411))
+- Email address
+- A phone number with access to SMS messages OR a secondary device (to recieve 2FA codes)
```
Comment (Macleykun) [inline guides/create-account.md:13]: I admit it's cleaner how you wrote it :)
But, perhaps it's wise to still include the reason why we need to recieve the 2FA codes? (aka the error you would get on linux-altserver)?


From testing i noticed linux-altserver only seems to work when adding a second device.
I assume for sideserver users sms only works?

If sms only works for sideserver, we could mention that the second device/logging in with your burner account is only needed if you use the linux methode of sideloading.

Comment (Macleykun) [reply guides/create-account.md:13]: Sorry i only now just noticed the NOTE: part...
Ehm, scrap that part of my comment :P you think it's wise to include the error code for the reason why we they need to recieve 2fa codes?

Comment (Swifticul) [reply guides/create-account.md:13]: Potentially, yeah! Although the issue might differ depending on the person. We should ask people to take this for a test drive and see if they encounter issues with SMS on Linux.

**Comment by Macleykun** [comment]
Mm, I did test it a lot (I made altcon which is used for Linux :)) and stumbled on this a few timesOp 15 apr. 2023 om 19:15 heeft swifticul ***@***.***> het volgende geschreven:﻿
@Swifticul commented on this pull request.



In guides/create-account.md:
>  
-- Emailadres (you can use: https://temp-mail.org/)
-- Phone number (use one you can actually keep recieving SMS codes for)
-- An iDevice (To avoid: This action cannot be completed at this time (-22411))
+- Email address
+- A phone number with access to SMS messages OR a secondary device (to recieve 2FA codes)

Potentially, yeah! Although the issue might differ depending on the person. We should ask people to take this for a test drive and see if they encounter issues with SMS on Linux.

File: guides/create-account.md (hunk 1 above)
Comment (Macleykun) [reply guides/create-account.md:13]: Mm, I did test it a lot (I made altcon which is used for Linux :)) and stumbled on this a few times.
Never got an SMS myself sadly :(

**Comment by Macleykun** [comment]
thanks fam :) <3

**Pull request merged**